pkg/transformer/utils.go

```diff
@@ -159,3 +161,60 @@ func Print(name, path string, trailing string, data []byte, toStdout, generateJS
 	}
 	return file
 }
+
+// getComposeFileDir returns compose file directory
+func GetComposeFileDir(inputFiles []string) (string, error) {
+	// Lets assume all the docker-compose files are in the same directory
+	inputFile := inputFiles[0]
+	if strings.Index(inputFile, "/") != 0 {
+		workDir, err := os.Getwd()
+		if err != nil {
+			return "", err
+		}
+		inputFile = filepath.Join(workDir, inputFile)
+	}
+	return filepath.Dir(inputFile), nil
+}
+
+// NewCommand TODO: comment
+func NewCommand(cmd string) *exec.Cmd {
+	return exec.Command("sh", "-c", cmd)
+}
+
+type Command struct {
+	Cmd string
+	Dir string
+}
+
+// Execute: Wrapper to execute shell commands
+func Execute(command Command) (output string, err error) {
```

Comment:
I'm not sure if putting this into transformer package is right. 
Wouldn't be better to abstract command execution to separate package/lib?